shaders/shader.frag

```diff
@@ -0,0 +1,22 @@
+#version 410
+
+const vec4 lightPos = vec4(0.0, 2.0, 2.0, 1.0);
+const vec4 color = vec4(0.5, 0.5, 0.5, 1.0);
+
+in vec4 pos;
+//in vec4 norm;
+in vec2 TexCoord;
+
+out vec4 fragColor;
+uniform sampler2D myTextureSampler;
+
+void main() {
+    //float ambient = 0.1;
```

Comment:
remove un-used code

Reply:
can you add a comment about what it does and why it's left in there?
i.e `#phong-shading code in case we need to add it back`

Reply:
Done!